.github/workflows/release.yml

```
@@ -0,0 +1,66 @@
+# This workflow will build deb binary
+
+name: Build deb binary
+
+on:
+  release:
+    types: [published]
+
+  workflow_dispatch:
+
+jobs:
+  get-ref-name:
+    runs-on: ubuntu-latest
+    outputs:
+      ref_name: ${{ steps.get-ref-name.outputs.ref_name }}
+    steps:
+      - name: Get ref_name
+        id: get-ref-name
+        run: |
+          if [ "${{ github.event_name }}" == "pull_request" ]; then
+            sanitized_ref=$(echo ${{ github.head_ref }} | sed 's/[^a-zA-Z0-9.]/-/g')
+          else
+            sanitized_ref=$(echo ${{ github.ref_name }} | sed 's/[^a-zA-Z0-9.]/-/g')
+          fi
+          echo "ref_name=$sanitized_ref"
+          echo "ref_name=$sanitized_ref" >> $GITHUB_OUTPUT || exit 1
+  build:
+    needs: get-ref-name
+    name: Build deb
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```

v3 is EOL soon I think